conform-pr/main.go

```diff
@@ -55,8 +55,13 @@ func runChecks(action *githubactions.Action) []error {
 
 // checkTitle checks if PR's title does not end with dot.
 func checkTitle(title string) error {
-	if strings.HasSuffix(title, ".") {
-		return fmt.Errorf("checkTitle: PR title must not end with dot, but it does")
+	match, err := regexp.MatchString(`[a-zA-Z0-9]$`, title)
```

Comment:
we use to add `` for code quotes.

Reply:
… in Markdown. That's not a Markdown file